doc/tutorial.rst

```diff
@@ -102,14 +102,14 @@ Poisson Log-likelihood
 The likelihood of observing the spike count :math:`y_i` under the Poisson
 likelihood function with inhomogeneous rate :math:`\lambda_i` is given by:
 
-.. math::    \prod_i P(y = y_i) = \prod_i \frac{e^{-\lambda_i} \lambda_i^{y_i}}{y_i!}
+.. math::    \prod_i P(y = y_i | X) = \prod_i \frac{e^{-\lambda_i} \lambda_i^{y_i}}{y_i!}
```

Comment:
should we also update the log likelihood `\mathcal{L}` to be `matcal{L}(y_i | X)` ? In our notation, it's often confusing to read because A is a function of B and B is a function of C but the thing we're trying to optimize is C. Explicitly noting what the quantity is a function of (at least X and y) might help readability.

Reply:
Hmm, I think we say quite clearly that we minimize the log-likelihood wrt `\beta_0` and `\beta`